src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -1418,7 +1425,7 @@ private void HandleNodeShutdown(int node, NodeShutdown shutdownPacket)
                     if (shutdownPacket.Exception != null)
                     {
                         ILoggingService loggingService = ((IBuildComponentHost)this).GetComponent(BuildComponentType.LoggingService) as ILoggingService;
-                        loggingService.LogError(BuildEventContext.Invalid, new BuildEventFileInfo(String.Empty) /* no project file */, "ChildExitedPrematurely", shutdownPacket.Exception.ToString(), ExceptionHandling.DebugDumpPath);
+                        loggingService.LogError(BuildEventContext.Invalid, new BuildEventFileInfo(String.Empty) /* no project file */, "ChildExitedPrematurely", node, ExceptionHandling.DebugDumpPath, shutdownPacket.Exception.ToString());
```

Comment:
Was this just horribly wrong before? It sure looks better now . . .

Reply:
Yes, but the other code path was generally or always what happened so maybe nobody noticed.